src/utils/amplify/amplifyClientUtils.ts

```diff
@@ -19,7 +19,7 @@ export const configureClientAmplify = () => {
               scopes: ['openid'],
               responseType: 'code',
               redirectSignIn: [CLIENT_BASE_URL + '/login'],
-              redirectSignOut: [CLIENT_BASE_URL, `https://${OAUTH_DOMAIN}`],
+              redirectSignOut: [`${CLIENT_BASE_URL}/login`, `https://${OAUTH_DOMAIN}`],
```

Comment:
Could you rewrite the URL for SignIn using the template literal, as well?
Also, could you remove the `https://${OAUTH_DOMAIN}` from the SignOut and check if it still works? Because it may be unnecessary.

Same for the `amplifyServerUtils.ts`.